aries/aries_vcx_core/src/anoncreds/anoncreds/mod.rs

```diff
@@ -1460,39 +1448,25 @@ impl BaseAnonCreds for Anoncreds {
             );
         }
 
-        let res_rev_reg_delta = res_rev_reg_delta
-            .ok()
-            .as_ref()
-            .map(serde_json::to_string)
-            .transpose();
-
-        if let Err(err) = &res_rev_reg_delta {
-            warn!(
-                "get_rev_reg_delta >> Unable to deserialize rev_reg_delta cache for rev_reg_id: \
-                 {}, error: {}",
-                rev_reg_id, err
-            );
-        }
-
-        Ok(res_rev_reg_delta.ok().flatten())
```

Comment:
love to see it gone

Reply:
Me too :) This is just the beginning (hopefully)